### DISCUSSION TITLE
What need to be done to display rtl language (such as Arabic or Hebrew)

### DISCUSSION THREAD

**Discussion by eliargon** [danpros/htmly#808] [General]
What need to be done to display rtl language (such as Arabic or Hebrew)
there is an Arabic language file
but no explaination of how to display text in that language from Right to Left.
Also: is it possible to set a language/alignment for a specific post?

**Comment by danpros** [comment]
Hello, we just need to modify the theme (mostly the CSS), no need to modify the core. Copy one of the theme and rename it eg. "my-theme", put it back in `themes` folder and enable it via admin panel, with this step your modification will still intact even when we updating the core.
